azure-templates/deploy-steps.yml

```diff
@@ -98,6 +124,17 @@ steps:
         deployToSlotOrASE: true
         slotName: 'staging'
       displayName: Deploy to staging slot
+
+    - template: templates/rest-healthcheck/template.yaml@pagopa-commons 
+      parameters:
+        azureSubscription: '${{ parameters.azureSubscription }}'
+        appName: '${{ parameters.appName }}'
+        endpoint: '${{ parameters.healthcheckEndpoint }}'
+        containerInstanceResourceGroup: '${{ parameters.containerInstanceResourceGroup }}'
+        containerInstanceVNet: '${{ parameters.containerInstanceVNet }}'
+        containerInstanceSubnet: '${{ parameters.containerInstanceSubnet }}'
+        containerInstanceCpu: '${{ parameters.containerInstanceCpu }}'
+        containerInstanceMemory: '${{ parameters.containerInstanceMemory }}'
```

Comment:
default value for `rest-healthcheck/template.yaml` we can remove it?

Reply:
ok